x/authenticator/ante/ante.go

```diff
@@ -162,18 +170,24 @@ func (ad AuthenticatorDecorator) AnteHandle(
 			tracks = append(tracks, func() error {
 				err := a11r.Track(cacheCtx, account, feePayer, msg, uint64(msgIndex), stringId)
 
+				// track should not fail in normal circumstances, since it is intended to update track state before execution.
+				// If it does fail, we log the error.
+				telemetry.IncrCounter(1, types.CounterKeyTrackFailed)
+				ad.authenticatorKeeper.Logger(ctx).Error(
```

Comment:
This seems to be happening regardless of the value of err

Reply:
That is intentional since track should never fail, track fail should mean something is wrong and we should be notified.

Reply:
oh I get what you mean now, will fix